Content.Server/Armor/ArmorSystem.cs

```diff
@@ -16,8 +17,29 @@ public override void Initialize()
 
             SubscribeLocalEvent<ArmorComponent, DamageModifyEvent>(OnDamageModify);
             SubscribeLocalEvent<ArmorComponent, GetVerbsEvent<ExamineVerb>>(OnArmorVerbExamine);
+            SubscribeLocalEvent<ArmorComponent, ExamineGroupEvent>(OnExamineStats);
         }
 
+        private void OnExamineStats(EntityUid uid, ArmorComponent component, ExamineGroupEvent args)
+        {
+            if (args.ExamineGroup != component.ExamineGroup)
+                return;
+            foreach (var coefficientArmor in component.Modifiers.Coefficients)
+            {
+                args.Entries.Add(new ExamineEntry(component.ExaminePriorityCoefficient,Loc.GetString("armor-coefficient-value",
+                    ("type", coefficientArmor.Key),
+                    ("value", MathF.Round((1f - coefficientArmor.Value) * 100, 1))
+                    )));
+            }
+
+            foreach (var flatArmor in component.Modifiers.FlatReduction)
+            {
+                args.Entries.Add(new ExamineEntry(component.ExaminePriorityFlat,Loc.GetString("armor-reduction-value",
+                    ("type", flatArmor.Key),
+                    ("value", flatArmor.Value)
+                    )));
+            }
+        }
         private void OnDamageModify(EntityUid uid, ArmorComponent component, DamageModifyEvent args)
```

Comment:
```suggestion
        }
        
        private void OnDamageModify(EntityUid uid, ArmorComponent component, DamageModifyEvent args)
```

Lots of missing newlines, please fix all of them.